app/docs/1.7.x/policies/health-check.md

```diff
@@ -3,7 +3,7 @@ title: Health Check
 ---
 
 {% tip %}
-Circuit Breaker is an outbound policy. Dataplanes whose configuration is modified are in the `sources` matcher.
+Health Check is an outbound policy. Dataplanes whose configuration is modified are in the `sources` matcher.
```

Comment:
I changed this because it was obviously wrong. This is not related to the PR but seems small enough so I thought I just include it here.

Reply:
Related enough